pkg/results/report.go

```diff
@@ -146,9 +146,63 @@ func (r *reporter) report(request Request) {
 		logrus.Tracef("could not create report request: %v", err)
 		return
 	}
+	sendRequest(req, r.client, r.username, r.password)
+}
+
+type PodScalerRequest struct {
+	workloadName     string
+	configuredMemory string
+	determinedMemory string
+}
+
+type PodScalerReporter struct {
+	client             *http.Client
+	username, password string
+	address            string
+}
+
+func (o *Options) PodScalerReporter() (*PodScalerReporter, error) {
+	username, password, err := getUsernameAndPassword(o.credentials)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get username and password: %w", err)
+	}
+
+	return &PodScalerReporter{
+		client:   &http.Client{},
+		username: username,
+		password: password,
+		address:  o.address,
+	}, nil
+}
+
+// ReportMemoryConfigurationWarning is used to send the information about memory configuration
+// from pod-scaler-admission to result-aggregator.
+func (r *PodScalerReporter) ReportMemoryConfigurationWarning(workflowName, configuredMemory, determinedMemory string) {
```

Comment:
This method can be unit tested. See: https://github.com/openshift/ci-tools/blob/2ffb68b53d9940bd1afa5ac84f38cb6c001ea50c/pkg/results/report_test.go#L25-L105